src/components/MDX/InlineCode.tsx

```diff
@@ -18,7 +18,7 @@ function InlineCode({
       className={cn(
         'inline text-code text-secondary dark:text-secondary-dark px-1 rounded-md no-underline',
         {
-          'bg-gray-30 bg-opacity-10 py-px': !isLink,
+          'bg-opacity-10 py-px': !isLink,
```

Comment:
Instead of removing the background, can you update the `a > code` color to be `#045975` (blue-60) here instead:

https://github.com/reactjs/react.dev/blob/af0358f1d2eebd6f15916b04ef7e8b910247a94b/src/styles/index.css#L447-L450